os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_peripheral/ble_tizenrt_server.c

```diff
@@ -389,6 +389,11 @@ trble_conn_handle rtw_ble_server_get_conn_handle_by_address(uint8_t* mac)
     return conn_id;
 }
 
+trble_result_e rtw_ble_server_set_device_name(uint8_t* name)
+{
+    le_set_gap_param(GAP_PARAM_DEVICE_NAME, GAP_DEVICE_NAME_LEN, name);
+    return TRBLE_SUCCESS;
+}
```

Comment:
Could you correct the spacing here?

Reply:
How ? Do you mean add a new line after ```line 396```?

Reply:
Yes, you are right.